ansible/roles/operator-pipeline/templates/openshift/tasks/show-support-link-with-pr.yml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: create-support-link-for-pr
+spec:
+  params:
+    - name: env
+      description: Enviroment. One of [dev, qa, stage, production]
+    - name: type
+      description: Connect case type.
+      default: "CERT"
+    - name: source
+      description: Connect case source.
+      default: "tekton"
+    - name: cert_project_id
+      description: Pyxis certification project id.
+    - name: cert_project_type
+      description: Connect cert project type. Should be Operator Bundle Image for operator pipelines.
+      default: "Operator Bundle Image"
+    - name: pull_request_url
+      default: ""
+  results:
+    - name: comment
+  steps:
+    - name: create-support-link-for-pr
+      image: quay.io/redhat-isv/operator-pipelines-images
+      script: |
+        #! /usr/bin/env bash
+        set -e
+
+        LINK_ENV="$(params.env)"
+        # url parameters
+        LINK_TYPE=$(printf %s "$(params.type)")
```

Comment:
You could create those variables with `env` clause, like in here: https://github.com/redhat-openshift-ecosystem/operator-pipelines/blob/main/ansible/roles/operator-pipeline/templates/openshift/tasks/preflight.yml#L69